<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>taskList</class>
 <widget class="QWidget" name="taskList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="newTaskBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>新建任务</string>
   </property>
  </widget>
  <widget class="QListWidget" name="taskListWgt">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragDropMode::DragOnly</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::DropAction::CopyAction</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="taskListLbl">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>现有任务</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startWorkBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>任务开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="workListLbl">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>210</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>执行列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearTaskBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清空任务列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearWorkBtn">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清空执行列表</string>
   </property>
  </widget>
  <widget class="QLabel" name="workTimesLbl">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>210</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>执行次数</string>
   </property>
  </widget>
  <widget class="QListWidget" name="workListWgt">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>169</width>
     <height>192</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragDropMode::DragDrop</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::DropAction::MoveAction</enum>
   </property>
   <property name="gridSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QTableWidget" name="workTimesWgt">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>230</y>
     <width>71</width>
     <height>192</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>40</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>40</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
  </widget>
  <widget class="QListWidget" name="rubbishWgt">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>480</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragDropMode::DropOnly</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::DropAction::MoveAction</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
